pan_tilt_node:
  ros__parameters:
    pan_min: -1.57
    pan_max: 1.57
    tilt_min: -0.78
    tilt_max: 0.78
    speed: 0.5
